packages/netlify-cms-widget-boolean/src/BooleanControl.js

```diff
@@ -16,34 +21,42 @@ function BooleanBackground({ isActive, ...props }) {
 }
 
 export default class BooleanControl extends React.Component {
+  static propTypes = {
+    field: ImmutablePropTypes.map.isRequired,
+    onChange: PropTypes.func.isRequired,
+    classNameWrapper: PropTypes.string.isRequired,
+    setActiveStyle: PropTypes.func.isRequired,
+    setInactiveStyle: PropTypes.func.isRequired,
+    forID: PropTypes.string,
+    value: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    value: false,
+  }
+
   render() {
-    const { value, forID, onChange, classNameWrapper, setActiveStyle, setInactiveStyle } =
+    const { value, forID, onChange, classNameWrapper, setActiveStyle, setInactiveStyle, field } =
       this.props;
+
+    const prefix = field.get('prefix', false);
+    const suffix = field.get('suffix', false);
+
     return (
       <div className={classNameWrapper}>
-        <Toggle
-          id={forID}
-          active={value}
-          onChange={onChange}
-          onFocus={setActiveStyle}
-          onBlur={setInactiveStyle}
-          Background={BooleanBackground}
-        />
+        <div css={innerWrapper}>
+          {prefix && (<span>{prefix}&nbsp;</span>)}
+          <Toggle
+            id={forID}
+            active={value}
+            onChange={onChange}
+            onFocus={setActiveStyle}
+            onBlur={setInactiveStyle}
+            Background={BooleanBackground}
+          />
+          {suffix && (<span>&nbsp;{suffix}</span>)}
+        </div>
       </div>
     );
   }
 }
-
-BooleanControl.propTypes = {
-  field: ImmutablePropTypes.map.isRequired,
-  onChange: PropTypes.func.isRequired,
-  classNameWrapper: PropTypes.string.isRequired,
-  setActiveStyle: PropTypes.func.isRequired,
-  setInactiveStyle: PropTypes.func.isRequired,
-  forID: PropTypes.string,
-  value: PropTypes.bool,
-};
-
-BooleanControl.defaultProps = {
-  value: false,
-};
```

Comment:
any reason why these were moved as part of this specific PR ?

Reply:
To match with other widget for consistency.